src/UploadedFile.php

```diff
@@ -60,40 +60,42 @@ public function __construct($streamOrFile, $size, $errorStatus, $clientFilename
             $this->stream = new Stream($streamOrFile);
         }
 
-        if (! $this->file && ! $this->stream) {
-            if (! $streamOrFile instanceof StreamInterface) {
+        if (!$this->file && !$this->stream) {
+            if (!$streamOrFile instanceof StreamInterface) {
                 throw new InvalidArgumentException('Invalid stream or file provided for UploadedFile');
             }
+
             $this->stream = $streamOrFile;
         }
 
-        if (! is_int($size)) {
+        if (!is_int($size)) {
             throw new InvalidArgumentException('Invalid size provided for UploadedFile; must be an int');
         }
+
         $this->size = $size;
 
-        if (! is_int($errorStatus)
-            || 0 > $errorStatus
-            || 8 < $errorStatus
-        ) {
+        if (!is_int($errorStatus) || 0 > $errorStatus || 8 < $errorStatus) {
```

Comment:
I prefer multiple lines here, as it's far simpler to see and understand the specific conditions than when on the same line.